### DISCUSSION TITLE
ERR_ABORTED 404

### DISCUSSION THREAD

**Discussion by blazerpro1** [chatwoot/chatwoot#4399] [Chatwoot APIs] [answered]
Hello everyone, I'm new to Chatwoot and am experiencing an error when integrating chatwoot with my localhost website.

I have pasted the given messenger script in my body tag but I receive the following error in the console.
![asd](https://user-images.githubusercontent.com/62301603/161860551-0d4dbf56-fc39-4030-96a1-5789ebda1dd6.jpeg)

Please enlighten me further for I may have done a mistake from my end.

**Comment by vishnu-narayanan** [comment, marked as answer]
@blazerpro1 Could you verify your Heroku deployment? Your installation URL seems to give 404.

https://doc-livechat-2.herokuapp.com
![image](https://user-images.githubusercontent.com/3526167/161920836-803239b4-943d-43ad-bd38-71b0bc294c67.png)

**Comment by blazerpro1** [comment]
```
<script>
      (function(d,t) {
        var BASE_URL="https://doc-livechat-2.herokuapp.com";
        var g=d.createElement(t),s=d.getElementsByTagName(t)[0];
        g.src=BASE_URL+"/packs/js/sdk.js";
        g.defer = true;
        g.async = true;
        s.parentNode.insertBefore(g,s);
        g.onload=function(){
          window.chatwootSDK.run({
            websiteToken: 'xxx',
            baseUrl: BASE_URL
          })
        }
      })(document,"script");
</script>
```
   
 I have found the issue. My base URL is supposed to be "https://doc-live-chat-2.herokuapp.com". But for some reason, Chatwoot's script gave me the wrong base URL.

**Reply by vishnu-narayanan** [reply]
Did you configure the `FRONTEND_URL` environment variable?
